dev/releases/make_archives.py

```diff
@@ -54,70 +68,92 @@
     gapversion = get_makefile_var("GAP_BUILD_VERSION")
 except:
     error("make sure GAP has been compiled via './configure && make'")
+
 notice(f"Detected GAP version {gapversion}")
 
-if re.fullmatch( r"[1-9]+\.[0-9]+\.[0-9]+", gapversion) != None:
-    notice(f"--- THIS LOOKS LIKE A RELEASE ---")
+if re.fullmatch(r"[1-9]+\.[0-9]+\.[0-9]+", gapversion) != None:
+    notice("--- THIS LOOKS LIKE A RELEASE ---")
     pkg_tag = f"v{gapversion}"
 else:
-    notice(f"--- THIS LOOKS LIKE A NIGHTLY BUILD ---")
+    notice("--- THIS LOOKS LIKE A NIGHTLY BUILD ---")
     pkg_tag = "latest"
 
 
 # extract commit_date with format YYYY-MM-DD
 # TODO: is that really what we want? Or should it be the date this
 # script was run? Or for releases, perhaps use the TaggerDate of the
 # release tag (but then we need to find and process that tag)
-commit_date = subprocess.run(["git", "show", "-s", "--format=%as"],
-                             check=True, capture_output=True, text=True)
+commit_date = subprocess.run(
+    ["git", "show", "-s", "--format=%as"], check=True, capture_output=True, text=True
+)
```

Comment:
I don't mind much, but personally I find the new version harder to read than the old :-)